app/services/ProductService.py

```
@@ -0,0 +1,3 @@
+
+def get_product_reviews()-> Product:
```

Ruff (0.8.2)

Undefined name `Product`

(F821)